admin-console/src/main/webapp/src/pages/ApplicationPage.tsx

```diff
@@ -112,15 +112,17 @@ export const ApplicationPage: React.FunctionComponent = () => {
                                                 <TextInput
                                                     type='text'
                                                     defaultValue={ eventType.name }
-                                                    onChange={ () => handleChange }
+                                                    onChange={ handleChange }
                                                     id='name'
+                                                    name="name"
```

Comment:
This is important, as we are using the name to know what field to populate on the handleChange function